src/main/java/it/gov/pagopa/arc/config/SwaggerConfig.java

```diff
@@ -1,4 +1,4 @@
-package it.gov.pagopa.common.config;
+package it.gov.pagopa.arc.config;
```

Comment:
I would let this class inside the `common` package, because this class will not contain "arc" related logic, but common functionalities that could be copy/pasted in order to be reused in other projects.

Using an agnostic package (not related to a particular application) will simplify its maintenance